examples/roitberg/roitberg.py

```diff
@@ -279,15 +286,24 @@ def parse_args():
     * clean_work_dir bool
       If true, indicates that the working directory should be wiped before
       training.
+    * reload_model bool
+      If true, indicates that a model should be reloaded instead of training
+      a new one from scratch.
   """
   parser = argparse.ArgumentParser(description="Run ANI1 neural net training.",
                                    formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+
   parser.add_argument('-b', '--batch-size', type=int, default=1152,
                       help="training batch size")
   parser.add_argument('--featurization-batch-size', type=int, default=384,
                       help="featurization batch size")
   parser.add_argument('-e', '--max-search-epochs', type=int, default=100,
                       help="The maximum number of epochs to run w/o validation score improvement.")
+  parser.add_argument('--initial-learning-rate', type=float, default=0.001,
```

Comment:
does this take scientific notation? i.e. 1e-3

Reply:
I think it takes any float format that python supports...